package.json

```diff
@@ -44,8 +44,8 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.45.3",
-    "@remix-run/dev": "^2.10.2",
-    "@remix-run/testing": "^2.10.3",
+    "@remix-run/dev": "^2.13.1",
+    "@remix-run/testing": "^2.13.1",
```

Comment:
This doesn't look right, right? Looks like you accidentally downgraded these packages maybe?

Reply:
Upgraded `npm upgrade` will upgrade packages.